src/main/java/io/spokestack/spokestack/SpeechContext.java

```diff
@@ -157,6 +157,11 @@ public boolean isActive() {
      */
     public SpeechContext setActive(boolean value) {
         this.active = value;
+        if (value) {
```

Comment:
we talked about edge triggering here, was that not feasible?

Reply:
I can't think of a place where that'd cause a problem; looks like I misread the definition of the term.